deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -1646,6 +1646,16 @@ end}.
     {datatype, string}
 ]}.
 
+
+%% Whether to verify if this is the first time a node starts.
+%% When enabled, nodes will create a marker file on first startup
+%% and refuse to start if the marker exists but tables are empty.
+%%
+
+{mapping, "verify_initial_run", "rabbit.verify_initial_run", [
```

Comment:
Can we rename this to `prevent_startup_if_node_was_reset`? I have considered many names, such as

 * `track_initial_run_and_seeding`
 * `enforce_initialized_database`
 
 and a few others but the most specific option is the one that describes the end goal: prevent a previously reset node from booting.

Reply:
Yep, will do